manifests/fmops-qna-rag/storage-modules.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: opensearch
+path: git::https://github.com/awslabs/idf-modules.git//modules/storage/opensearch/
```

Comment:
We should lock this version to a release of IDF:
```
git::https://github.com/awslabs/idf-modules.git//modules/storage/opensearch?ref=release/1.7.0&depth=1
```

Reply:
Updated the manifest